### ISSUE TITLE
redacted: System halts when trying to add tracker that never saves

### ISSUE THREAD

**Issue by flying-sausages** [Jackett/Jackett#8901]
### Environment

**OS**: Ubuntu 20.04

**.Net Runtime**: Mono

**.Net Version**:
```
Mono JIT compiler version 6.8.0.105 (Debian 6.8.0.105+dfsg-2 Wed Feb 26 23:23:50 UTC 2020)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
	TLS:           __thread
	SIGSEGV:       altstack
	Notifications: epoll
	Architecture:  amd64
	Disabled:      none
	Misc:          softdebug 
	Interpreter:   yes
	LLVM:          supported, not enabled.
	Suspend:       hybrid
	GC:            sgen (concurrent by default)
```

**Jackett Version**: 0.16.616.0 

**Last Working Jackett Version**: ??

**Are you using a proxy or VPN?** [yes/no] no

### Description

When adding a definition for redacted.ch, the definition never saves, and the spinning wheel on top of the Okay button keeps spinning. Eventually the system becomes unresponsive and all other applications on the server halt. SSH sessions time out and KVM is unresponsive.

To reproduce, add redacted.ch tracker, use username, password, and cookie. This bug cannot be reproduced when a cookie is _not_ set.

### Logged Error Messages

Paste of the logs, which will expire within a month of posting. https://0bin.net/paste/BGgagHcoqjdVErh+#g5JJWZd4S8hLzF3koGYNuBTftq2Nia94oagKqGJFbQI

### Screenshots

[Place any screenshots of the issue here if needed]

**Comment by flying-sausages** [comment]
It might be easier to fix this by implementing #8898 ?

**Comment by ngosang** [comment]
By now you can disable 2FA. It works well.

**Comment by flying-sausages** [comment]
> By now you can disable 2FA. It works well.

Sorry I'm not sure I understand. Are you saying I should disable the 2FA on redacted?

**Comment by flying-sausages** [comment]
Jackett goes sideways whenever I set something in the cookie field, be it a valid or invalid cookie.

**Comment by seeyabye** [comment]
@garfield69 @ngosang Should we close this with the resolution of https://github.com/Jackett/Jackett/pull/9516?

**Comment by garfield69** [comment]
closed with PR #9666
